troposphere/ec2.py

```diff
@@ -317,6 +317,7 @@ class Route(AWSObject):
     props = {
         'DestinationCidrBlock': (basestring, False),
         'DestinationIpv6CidrBlock': (basestring, False),
+        'EgressOnlyInternetGatewayId': (basestring, False),
```

Comment:
Would you like to add additional validation? You can follow the same pattern for the Destination* props in the validate method. Per the docs:
```
Required: Conditional. You must specify only one of the following properties: EgressOnlyInternetGatewayId, GatewayId, InstanceId, NatGatewayId, NetworkInterfaceId, or VpcPeeringConnectionId.
```